src/uu/link/src/link.rs

```diff
@@ -1,3 +1,4 @@
+use clap::builder::ValueParser;
```

Comment:
oups, it should be after the copyright, please

Reply:
Sorry about that, I didn't notice it. It's fixed in https://github.com/uutils/coreutils/pull/3877